release-top-level-artifacts/README.md

```diff
@@ -102,11 +102,28 @@ Open `helm-charts/yunikorn/values.yaml` and replace the docker image information
 For more instructions, please refer to [User Guide](https://yunikorn.apache.org/docs/).
 
 ## Deploying a convenience build
-Apache YuniKorn provides a convenience release with pre-build docker images and helm charts.
+Apache YuniKorn provides a convenience release with pre-built Docker images and Helm charts.
 These can be accessed via the [downloads page](https://yunikorn.apache.org/community/download) and instructions are
 located in the [User Guide](https://yunikorn.apache.org/docs/).
 
-The convenience build images are multi architecture images. Supported architectures are  `amd64` and `arm64v8`.
+The convenience build images are multi-architecture images. Supported architectures are `amd64` and `arm64v8`.
```

Comment:
We should fix this now. We have removed the `v8` extension in an earlier release (1.3.0) and generate just `arm64` as the container spec has dropped the `v8` It is the default variant and gets removed as part of normalisation.

Reply:
Updated in latest commit.